LiskHQ/lisk-hub
# Code splitting by route - Closes #519

### What was the problem?
We have a big Lisk Hub js bundle that load all components for different routes

### How did I fix it?
Created `asyncComponent` that allows import component only then it needed.
### How to test it?
- Open network tab in dev inspector
- Click different routes and see that code chunks is loaded

Before code splitting:
<img width="720" alt="screen shot 2018-04-04 at 17 00 09" src="https://user-images.githubusercontent.com/922338/38313017-d415e286-382b-11e8-88cd-4f25802e1895.png">

After:
<img width="755" alt="screen shot 2018-04-04 at 16 56 38" src="https://user-images.githubusercontent.com/922338/38313033-e340d9a0-382b-11e8-9bdc-fb54cf72d204.png">



### Review checklist
- The PR solves #519 
- All new code follows best practices
